testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -746,7 +746,32 @@ public void testSelectCaseInsensitive()
     @Test
     public void testTopN()
     {
-        assertQuery("SELECT n.name, r.name FROM nation n LEFT JOIN region r ON n.regionkey = r.regionkey ORDER BY n.name LIMIT 1");
```

Comment:
Let's keep the original short test method. 
It doesn't make sense to put the connector under the test _if_ it does not support TopN pushdown (and most do not today).

Ideally, we should remove this test from here and into Engine-only queries (do we have it?)
and have a BCT test that verified that either connector does not support TopN pushdown or is thoroughly tested, like we do with eg inserts.